interfaces/builtin/pulseaudio.go

```diff
@@ -31,6 +31,7 @@ const pulseaudioConnectedPlugAppArmor = `
 
 owner /{,var/}run/pulse/ r,
 owner /{,var/}run/pulse/native rwk,
+owner /run/user/[0-9]*/{,pulse/} rw,
```

Comment:
This rule isn't quite right. You want this instead:
```
owner /run/user/[0-9]*/ r,
owner /run/user/[0-9]*/pulse/ rw,
```

A snap that 'plugs: [ pulseaudio ]' should not be able to manipulate '/run/user/[0-9]*/'. Something else should be creating '/run/user/[0-9]*/' (in fact, snap-confine will when is supports XDG_RUNTIME_DIR, which is pending).